src/login/LoginFailure.jsx

```diff
@@ -68,6 +70,24 @@ const LoginFailureMessage = (props) => {
       );
       break;
     }
+    case ALLOWED_DOMAIN_LOGIN_ERROR: {
+      const url = `${getConfig().LMS_BASE_URL}/dashboard/?tpa_hint=${context.tpa_hint}`;
```

Comment:
The values are changed to camelcase in the response on authn. 
```suggestion
      const url = `${getConfig().LMS_BASE_URL}/dashboard/?tpa_hint=${context.tpaHint}`;
```